presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -177,8 +178,8 @@ public DistributedQueryRunner build()
                         .put("hive.storage-format", "TEXTFILE") // so that there's no minimum split size for the file
                         .put("hive.compression-codec", "NONE") // so that the file is splittable
                         .build();
-                queryRunner.createCatalog(HIVE_CATALOG, HIVE_CATALOG, hiveProperties);
-                queryRunner.createCatalog(HIVE_BUCKETED_CATALOG, HIVE_CATALOG, hiveBucketedProperties);
+                queryRunner.createCatalog(HIVE_CATALOG, HIVE_CONNECTOR_NAME, hiveProperties);
```

Comment:
good catch;

there is a  problem though that this is _not_ actually "hive connector's name" (the latter being `hive-hadoop2`). It's `TestingHiveConnectorFactory`'s name.

 you can inline the constant avoiding the naming problem